style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+#title {
+  font-size: 1.8rem;
+  font-family: sans-serif;
+  margin: 4rem auto;
+  text-align: center;
+}
+
+body {
+
+  background-color: greenyellow;
+  color: rgb(39, 36, 36);
+  width: 100%;
+  height: auto;
+}
+
+#survey-form {
+  padding: 3rem 2.5rem;
+  border-radius: 7px;
+}
+
+#submit {
```

Comment:
use class selector instead of using id

Reply:
- [ ] [OPTIONAL] Great job giving reasons for the requested change. However, I suggest that you also phrase your comments politely and use suggestions.

### EXAMPLE
I Kindly suggest that you use a class selector instead of using an id

Reply:
- [ ]  [OPTIONAL] Great job giving reasons for the requested change. However, I suggest that you also phrase your comments politely and use suggestions.
EXAMPLE
I Kindly suggest that you use a class selector instead of using an id